stable/nuclio/templates/playground.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+{{- if .Values.Playground.Enabled }}
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "nuclio.playgroundName" . }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "nuclio.name" . }}
+        release: {{ .Release.Name }}
+        component: {{ template "nuclio.playgroundName" . }}
+    spec:
+      containers:
+      - name: {{ template "nuclio.playgroundName" . }}
+        image: {{ template "nuclio.playgroundImage" . }}
+        imagePullPolicy: {{ .Values.Playground.PullPolicy }}
+        ports:
+        - containerPort: 8070
+        volumeMounts:
+        - mountPath: /var/run/docker.sock
+          name: docker-sock
+      volumes:
+      - name: docker-sock
+        hostPath:
+          path: /var/run/docker.sock
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "nuclio.playgroundName" . }}
+spec:
+  selector:
+    app: {{ template "nuclio.name" . }}
+    release: {{ .Release.Name }}
+    component: {{ template "nuclio.playgroundName" . }}
+{{- if eq .Values.Playground.Service.Type "NodePort" }}
+  type: NodePort
+{{- end }}
+  ports:
+  - name: {{ template "nuclio.playgroundName" . }}
```

Comment:
Why use a template for ports? Use something short, such as `web` or `http`, whatever makes sense.

Reply:
changed to `playground`